pallets/runtime/tests/src/staking/mod.rs

```diff
@@ -37,6 +37,33 @@ macro_rules! assert_session_era {
     };
 }
 
+macro_rules! assert_present_entity {
+    ($acc_id:expr) => {
+        assert_eq!(
+            Staking::permissioned_entities(Identity::get_identity($acc_id).unwrap()),
+            true
+        );
+    };
+}
+
+macro_rules! assert_absent_entity {
+    ($acc_id:expr) => {
+        assert_eq!(
+            Staking::permissioned_entities(Identity::get_identity($acc_id).unwrap()),
+            false
+        );
+    };
+}
+
+macro_rules! assert_add_permissioned_validator {
+    ($acc_id:expr) => {
+        assert_ok!(Staking::add_permissioned_validator_entity(
+            frame_system::RawOrigin::Root.into(),
```

Comment:
```suggestion
            root(),
```
(`use super::committee_test::root`)

Reply:
It doesn't give anything extra?? Although It is better to move `root()` from `committee_test` to storage.rs.

Reply:
No, its just shorter; Feel free to move it though.